my-app/src/components/calendar/calendar.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './calendar.css'; 
+import FullCalendar from '@fullcalendar/react'
+import dayGridPlugin from '@fullcalendar/daygrid'
+import timeGridPlugin from '@fullcalendar/timegrid';
+
+let styles = {
+    marginTop: '50px',
+    marginLeft: '65px',
+    maxWidth: '900px',
+  };
+  
+
+
+class CalendarPage extends React.Component {
```

Comment:
The Calendar looks good, my only suggestion is to see if you can have it centered on the page, or see if you can stretch it so it fills up more of the screen.

Reply:
Ya that's an easy fix. Just had it like that because I was thinking of adding other search menus on the side